docs/CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+
+> This CHANGELOG is a bit different; we are only adding the versions for the first time.
+> In the Breaking Changes section, we’ve listed the versions that have been changed.
+> In the New Features section, we’ve listed the versions that didn’t exist before.
+
+## [v11.0.0] - 2025-07-21 (First ChangeLog)
+
+### 🚨 Breaking Changes
```

Comment:
are these really breaking changes?

Reply:
Probably not! But I think it's important to emphasize that we've changed the version so that everyone knows.